### PR TITLE
add macOS support

### PR DESCRIPTION
**Pull request by sfroment**
add basic macOS support

### CODE REVIEW

**Comment by adi1090x** [comment]
you tested it on mac...did it work?
i can't  because i don't use mac.

**Comment by sfroment** [comment]
Yup ! 
out

**Comment by sfroment** [comment]
If #2 is merged when it worked, there's no problem that I update my PR to follow a more readable way to do it

File: battery_wall.sh [hunk 1]
```diff
@@ -2,33 +2,51 @@
 
 ## Author : Aditya Shakya (adi1090x)
 
-DIR="/usr/share/battery-wallpaper"
+case "$OSTYPE" in
+	darwin*) DIR="/usr/local/share/battery-wallpaper" ;;
+	linux*) DIR="/usr/local/battery-wallpaper" ;;
```
Comment (adi1090x) [inline battery_wall.sh]: it should be ***/usr/share/battery-wallpaper*** according to changes you made in install.sh

**Review by adi1090x — COMMENTED** [review]
No, its okay... it's good enough.
can you just review the line-7 in battery_wall.sh in your PR.

**Comment by sfroment** [comment]
:+1: updated

**Comment by adi1090x** [comment]
Great, merging it.

**Comment by adi1090x** [comment]
Hey @sfroment , I've updated the code. can you try it on MAC?
I'm sure it'll not work on macos, so fix it if you can. thanks.

**Comment by sfroment** [comment]
I'll try it ASAP

**Pull request merged**